boutique-eshop-yugabyte/yugabyte/aws.yaml

```diff
@@ -15,7 +15,7 @@ storage:
 replicas:
   master: 1
   tserver: 1
-  totalMasters: 3
+  totalMasters: 4
```

Comment:
In real life it does not make sense to have 4 masters, but I guess to prove the point is fine